src/pybind/mgr/rbd_support/mirror_snapshot_schedule.py

```diff
@@ -389,11 +414,12 @@ def get_ioctx(self, image_spec):
                 ioctx.set_namespace(namespace)
                 images = set()
                 self.ioctxs[nspec] = (ioctx, images)
+            assert images
```

Comment:
I observe this when running `cli_generic.sh`:
```
2021-02-11T20:17:59.998+0000 7f169e740640  0 [rbd_support CRITICAL root] Fatal runtime error: ImageSpec(pool_id='19', namespace='ns1', image_id='79e7bf3d9a72')
Traceback (most recent call last):
  File "/home/mgolub/ceph/ceph.ci/src/pybind/mgr/rbd_support/mirror_snapshot_schedule.py", line 206, in open_image
    ioctx = self.get_ioctx(image_spec)
  File "/home/mgolub/ceph/ceph.ci/src/pybind/mgr/rbd_support/mirror_snapshot_schedule.py", line 417, in get_ioctx
    assert images
AssertionError
```
which I believe fails for empty set, which is quite possible here.